w3-filecoin.md

```diff
@@ -304,265 +355,267 @@ If the added piece is invalid, details on failing reason is also reported:
 }
 ```
 
-### Aggregator Capabilities
+### _Aggregator_ Capabilities
 
-#### `aggregate/queue`
+#### `piece/offer`
 
-A Storefront can invoke a capability to offer a piece to be aggregated for upcoming Filecoin deal(s). See [schema](#aggregateadd-schema).
+A [Storefront] can invoke a capability to offer a piece to be aggregated for upcoming Filecoin deal(s). See [schema](#pieceoffer-schema).
 
-> `did:web:web3.storage` invokes capability from `did:key:agg...`
+> `did:web:web3.storage` invokes capability from `did:web:aggregator.web3.storage`
 
 ```json
 {
   "iss": "did:web:web3.storage",
-  "aud": "did:key:agg...",
-  "att": [{
-    "with": "did:web:web3.storage",
-    "can": "aggregate/queue",
-    "nb": {
-      "piece": { "/": "bafk...commp" }, /* commitment proof for piece */
-      "group": "did:web:free.web3.storage", /* grouping of joining segments into an aggregate */
+  "aud": "did:web:aggregator.web3.storage",
+  "att": [
+    {
+      "with": "did:web:web3.storage",
+      "can": "piece/offer",
+      "nb": {
+        /* commitment proof for piece */
+        "piece": { "/": "bafk...commp" },
+        /* grouping of joining segments into an aggregate */
+        "group": "did:web:free.web3.storage"
+      }
     }
-  }],
+  ],
   "prf": [],
   "sig": "..."
 }
 ```
 
-An Aggregator MUST issue a signed receipt to acknowledge the received request. The issued receipt MUST contain an [effect](https://github.com/ucan-wg/invocation/#7-effect) with a subsequent task (`.fx.join` field) that is run when piece is added to an aggregate and either succeeds (implying that aggregate was queued for being offered) or fails (with an `error` describing the problem).
+An [Aggregator] MUST issue a signed receipt to acknowledge the received request. The receipt MUST contain a `fx.join` [effect] linking to [`piece/accept`] task that MUST either succeed with [`InclusionProof`] or fail with an error describing the reason.
```

Comment:
```suggestion
An [Aggregator] MUST issue a signed receipt to acknowledge the received request. The receipt MUST contain an `fx.join` [effect] linking to [`piece/accept`] task that MUST either succeed with an [`InclusionProof`] or fail with an error describing the reason.
```

Reply:
grammar plugin I instal insisted that "a" is correct given `fx.join` 😭